plugins/org.jboss.tools.vpe.preview/src/org/jboss/tools/vpe/preview/view/VpvView.java

```diff
@@ -263,6 +264,21 @@ public IStatus runInUIThread(IProgressMonitor monitor) {
 		return job;
 	}
 
+	private Job createSaveJob() {
+		Job job = new UIJob("Saving") { //$NON-NLS-1$
```

Comment:
Lets show who requested the saving and name the job instance properly using this template:
HTML Preview: Saving '${filename-without-path}'
I borrowed this format from system job name which looks like:
System: Message text